CrossCompatibility/CrossCompatibility.psd1

```diff
@@ -0,0 +1,123 @@
+#
+# Module manifest for module 'CrossCompatibility'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 11/5/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'CrossCompatibility.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '84f45c56-1fc4-4253-af1a-9ef78cbd4dbc'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) Microsoft Corporation'
+
+# Description of the functionality provided by this module
+Description = 'Collects and makes available information on PowerShell runtimes across platforms'
+
+# Minimum version of the PowerShell engine required by this module
+PowerShellVersion = '3.0'
+
+# Name of the PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+# RequiredModules = @()
+
+# Assemblies that must be loaded prior to importing this module
+# RequiredAssemblies = @()
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+# FormatsToProcess = @()
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+# NestedModules = @()
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = '*'

```

Comment:
Yeah, I deferred deciding what to export here since it's not so important. I think the best plan is to talk through the functions and work out what level of granularity to go with, although I'd be quite happy just exporting the whole list of things.

Reply:
Yes, I agree that using `*` does not make a good impression in this repo
https://github.com/PowerShell/PSScriptAnalyzer/blob/development/RuleDocumentation/UseToExportFieldsInManifest.md